requirements-build.txt

```diff
@@ -1 +1 @@
-pyinstaller==3.4
+git+git://github.com/pyinstaller/pyinstaller@4f051b301b0ed0a79b8605a07719222c19459e3d#egg=pyinstaller
```

Comment:
What about just bumping `pyinstaller` version to 3.5?

Reply:
At the moment I created the PR 3.5 wasn't released yet. It's good to know they finally did another release. I will bump it accordingly.